docs/source/networks.rst

```diff
@@ -0,0 +1,311 @@
+source, it will be used as the parent for all first generation nodes. After
+the first generation, the parent from each new node will be selected from the
+previous generation, using the `fitness` attribute of the nodes to select it.
+The higher the fitness, the higher the probability that a node will be a
+parent.
+
+Note that for this network to function correctly, the experiment nodes need
+to have a `generation` property defined.
+
+ScaleFree
+^^^^^^^^^
+
+This network takes two parameters: `m0` and `m`. The first (m0) is the
+number of initial nodes. These initial nodes will be connected in a fully
+connected network among each other. The second parameter (m) is the number of
+connections that every subsequent node will have. The nodes for this limited
+number of connections will be chosen randomly, but nodes with more
+connections will have a higher probability of being selected.
+
+SequentialMicrosociety
+^^^^^^^^^^^^^^^^^^^^^^
+
+A network in which each new node will be connected using a `to` connection to
+a limited set of its most recent predecessors. The number of recent
+predecessors is passed in as an argument (n) at network creation.
+
+SplitSampleNetwork
+^^^^^^^^^^^^^^^^^^
+
+This network helps when implementing split sample experiment designs. It
+assigns a random boolean value to a property named `exploratory`. When this
+property is True, it means that the current network is part of the
+exploratory data subset.
+
+Creating a network
+------------------
+
+In addition to the available networks, it's fairly simple to create a custom
+network, in case an experiment design calls for different node
+interconnections. To create one, we can subclass from the Network model:
+
+::
+
+    from dallinger.models import Network
+    from dallinger.nodes import Source
+
+
+    class Ring(Network):
+
+        __mapper_args__ = {"polymorphic_identity": "ring"}
+
+        def add_node(self, node):
+            other_nodes = [n for n in self.nodes() if n.id != node.id]
+
+            if isinstance(node, Source):
+                raise Exception(
+                    "Ring network cannot contain sources."
+                )
+
+            if other_nodes:
+                parent = max(other_nodes, key=attrgetter('creation_time'))
+                parent.connect(whom=node)
+
+                if len(self.nodes) == self.max_size:
+                    parent = min(other_nodes, key=attrgetter('creation_time'))
+                    node.connect(whom=parent)
+                   
+In the above example, we create a simple `ring` network, where each node is
+connected in chain to the next one, until we get to the last one, which is
+connected back to the first, making a full circle (thus, the ring name).
+
+Ring is a subclass of `dallinger.models.Network`, which contains the basic
+network model and implementation. The `__mapper_args__` assignment at the
+top is for differentiating this network from the rest. Usually the safe
```

Comment:
I think you should say it's used for differentiating it in data exports. Not having this set to a unique value will cause data exports to be incorrect.